tests/build/multi-platform.go

```diff
@@ -59,7 +59,7 @@ var (
 	interval                     = 10 * time.Second
 )
 
-var _ = framework.MultiPlatformBuildSuiteDescribe("Multi Platform Controller E2E tests", Pending, Label("multi-platform"), func() {
+var _ = framework.MultiPlatformBuildSuiteDescribe("Multi Platform Controller E2E tests", Pending, Label("build-service", "multi-platform"), func() {
```

Comment:
I think these tests should be running in [this repo](https://github.com/konflux-ci/multi-platform-controller) repo CI, can we change `build-service` label and maybe keep `multi-platform` only?

Reply:
agree, `multi-platform` label should be sufficient.